.github/workflows/integration-test.yml

```
@@ -0,0 +1,60 @@
+name: "integration test"
```

NIT: Keeps the check name format aligned
```suggestion
name: "Test service integration"
```